### PR TITLE
Simple method to mitigate the 'overconfidentness' of NN on unrelated data.

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

**Comment by AntoineTheb** [comment]
Over-confidence ?

File: article/_posts/2019-07-11-overconfident.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+While this problem cannot be solved per se, it can nonetheless be mitigated.
+
+# Methods
+
+The authors propose a method to enforce a uniform confidence for any point far away from the training data.  They do this with a so-called *Adversarial confidence enhanced training* (ACET)
+
+![](/article/images/overconfidence/sc03.jpg)
+
+where the first term is the usual cross-entropy and the second term is a penalty term based on random images adversarial images $$Z$$ drawn from a noise distribution.  Illustration of these images for the training of CIFAR10 is as follows:
```
Comment (jhlegarreta) [inline article/_posts/2019-07-11-overconfident.md]: *(...) on random images adversarial images $$Z$$*

Looks like the first `images` is not needed?

**Comment by jodoin** [comment]
> Over-confidence ?

Yes probably.  That is why I put the quotes.

File: article/_posts/2019-07-11-overconfident.md (hunk 1 above)
Comment (jhlegarreta) [reply article/_posts/2019-07-11-overconfident.md]: Fixed in 96dc6ad.

**Pull request merged**